app/shared/src/forms/datePicker/__tests__/DatePicker.test.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import * as React from 'react';
+import { PlaygroundRenderer } from '@kiwicom/react-native-app-playground';
+import { View } from 'react-native';
+import Translation from '@kiwicom/react-native-app-translations';
+import MockDate from 'mockdate';
+
+import DatePicker from '../DatePicker';
+import TextIcon from '../../../icons/TextIcon';
+
+MockDate.set('2018-01-01');
+
+class PickerWrapper extends React.Component<{}, { date: Date }> {
+  state = {
+    date: new Date(),
+  };
+
+  onDateChange = (date: Date) => {
+    this.setState({ date });
+  };
+
+  render = () => (
+    <DatePicker
+      date={this.state.date}
+      onDateChange={this.onDateChange}
+      iconComponent={<TextIcon>&#xe0a2;</TextIcon>}
+      {...this.props}
+    />
+  );
+}
+
+describe('DatePicker', () => {
+  it('renders in playground', () => {
+    PlaygroundRenderer.render(<PickerWrapper />);
+  });
+
+  it('works with min date', () => {
+    PlaygroundRenderer.render(
+      <View>
+        <Translation passThrough="With min date" />
```

Comment:
Please don't do this. Always think about this file as about the test file. This is absolutely not relevant to the test. We should use the description from `describe` or `it` if you would like to see the description in the Playground.